Automattic/wp-calypso
# Scheduled Republicize: update form params

This replaces the blacklist skip_connections with a whitelist connections to work with the recent updates to the /sites/:site/post/:post/publicize endpoint.

**Testing**

1. Open sharing form on a post
2. Deselect all but one sharing network connection
3. Verify that a connections parameter is sent with the request
4. Verify that skip_connections is not sent with the request
5. Verify that the message is posted to only the selection sharing network
